packages/atomic/src/components/insight/result-templates/atomic-insight-result-children/atomic-insight-result-children.tsx

```diff
@@ -74,6 +74,10 @@ export class AtomicResultChildren
    * The non-localized copy for an empty result state.
    */
   @Prop() public noResultText = 'no-documents-related';
+  /**
+   * Whether to show a message when no child results are found.
+   */
+  @Prop() public hideNoResults = false;
```

Comment:
I think we should reverse this and also copy the name of the property above: `showNoResultText = true`.

Alternatively, we could remove this property and decide not to show the text when `noResultText` is an empty string. I'm fine with either solution, they both have their pros and cons.

Reply:
Since working in Quantic I tend to avoid boolean props defaulting to true. This is the reason they give:

_"Boolean attributes on standard HTML Elements are set to true by adding the attribute to the element. The absence of the attribute defaults the attribute to false. Therefore, the default value of an attribute is always false. Lightning web components use the same principle for boolean properties."_ [Source](https://developer.salesforce.com/docs/component-library/documentation/en/lwc/js_props_boolean)

It makes sense but personally i'm not super conflicted by doing it here.

I had thought about the empty `noResultText` doing the same. I like that it doesn't add any props and it makes sense because otherwise you're just rendering useless empty space. IMO we just need to communicate it clearly in the prop documentation as it isn't exactly intuitive. The HTML might just look a little bit odd.
```
<atomic-result-children no-result-text=""></atomic-result-children>
```
If you're cool with it so am I.

Reply:
I'm cool with the empty string